site/docs/config.toml

```diff
@@ -58,6 +59,10 @@ pygmentsCodeFences = true
 pygmentsStyle = "tango"
 disableKinds = ["taxonomy", "taxonomyTerm", "RSS", "sitemap"]
 googleAnalytics = "UA-151030466-2"
+staticDir = [
+  # This is the generated software APIs
```

Comment:
nit: plural or singular :)?

Reply:
Plural - There are Host APIs and Device APIs.